examples/angular-cli/package.json

```diff
@@ -15,7 +15,8 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:generate-output": "jest --json --config=jest.addon-config.js --outputFile=addon-jest.testresults.json || true",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "postinstall": "ngcc --source ../../node_modules"
```

Comment:
Do we need to target every package? most of them don't need to be compiled with ngcc and this will decrease the Dx again

Have you had trouble with `--target %list-of-packages%`?

Reply:
ngcc doesn't compile all the dependencies but only the one related to angular, no ? 
I had test with a target selection but it did not work with test snapshot. but maybe my list was not complete / good 🤷‍♂️